client/apps/edit/components/section_video/test/index.coffee

```diff
@@ -22,6 +22,7 @@ describe 'SectionVideo', ->
       props = {
```

Comment:
Additionally you will notice the [global window](https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_image/test/index.coffee#L24) required just above on ln 24 and 25-- draft.js needs these declared in the test environment to run correctly